update README.md and add the way to set up websocket proxy. (#571)
adshao/go-binance: README.md

[web-socket-streams.md](https://github.com/binance/binance-spot-api-docs/blob/master/web-socket-streams.md) | Details on available streams and payloads | <input type="checkbox" checked>  Implemented
[user-data-stream.md](https://github.com/binance/binance-spot-api-docs/blob/master/user-data-stream.md) | Details on the dedicated account stream | <input type="checkbox" checked>  Implemented
[margin-api.md](https://binance-docs.github.io/apidocs/spot/en) | Details on the Margin API (/sapi) | <input type="checkbox" checked>  Implemented
[futures-api.md](https://binance-docs.github.io/apidocs/futures/en/#general-info) | Details on the Futures API (/fapi) | <input type="checkbox" checked>  Implemented
[delivery-api.md](https://binance-docs.github.io/apidocs/delivery/en/#general-info) | Details on the Coin-M Futures API (/dapi) | <input type="checkbox" checked>  Implemented
[eoptions-api.md](https://binance-docs.github.io/apidocs/voptions/en/#general-info) | Detains on the European Options API(/eapi) | <input type="checkbox" checked>  Implemented  

  

> Found an unimplemented interface, please submit a issue.

Reviewer: ```suggestion
If you find an unimplemented interface, please submit an issue.
```

```golang
import (
    // for spot and other interfaces contained in https://binance-docs.github.io/apidocs/spot/en/#change-log
    "github.com/adshao/go-binance/v2"
    
    "github.com/adshao/go-binance/v2/futures" // optional package
    "github.com/adshao/go-binance/v2/delivery" // optional package

>     "github.com/adshao/go-binance/v2/eoptions" // optional package

Reviewer: To ensure compatibility, it's better to use `options` as the package name instead of `eoptions`. We can rename `options` to `eoptions` when Binance introduces American options.

If you have any questions, please refer to the specific version of the code for specific reference definitions or usage methods

##### Proxy Client
  
```

> proxyUrl := "http://127.0.0.1:7890" // for example, please replace as your exact proxy url.

Reviewer: ```suggestion
proxyUrl := "http://127.0.0.1:7890" // Please replace it with your exact proxy URL.
```

You don't need Client in websocket API. Just call binance.WsXxxServe(args, handler, errHandler).

> > For delivery API you can use `delivery.WsXxxServe(args, handler, errHandler)`.  

Reviewer: ```suggestion
> For delivery API you can use `delivery.WsXxxServe(args, handler, errHandler)`.
```

> If you wanna use proxy, you can set `HTTPS_PROXY` or `HTTP_PROXY` in environment variable or you can call `SetWsProxyUrl` of target packages in code. Then you can call other websocket function. For example below:

Reviewer: ```suggestion
If you want to use a proxy, you can set `HTTPS_PROXY` or `HTTP_PROXY` in the environment variable, or you can call `SetWsProxyUrl` in the target packages within your code. Then you can call other websocket functions. For example:
```
